enterprise_access/apps/enterprise_groups/management/commands/groups_reminder_emails.py

```diff
@@ -0,0 +1,58 @@
+"""
+Django management command for sending reminder emails to pending users to accept groups invitation.
+"""
+
+import logging
+
+from django.core.management import BaseCommand
+
+from enterprise_access.apps.api_client.enterprise_catalog_client import EnterpriseCatalogApiClient
+from enterprise_access.apps.api_client.lms_client import LmsApiClient
+from enterprise_access.apps.enterprise_groups.tasks import send_group_reminder_emails
+from enterprise_access.apps.subsidy_access_policy.models import PolicyGroupAssociation
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Django management command for sending reminder emails to pending users that have been added to a group.
+
+    This command sends reminder emails to learners at the 5, 25, 50, 65, and 85 day mark before the 90
+    day purge date.
+
+    """
+
+    help = "Send auto reminder emails to pending enterprise customer users that have been added to a group."
+
+    def handle(self, *args, **options):
+        """
+        Command's entry point.
+        """
+        LOGGER.info("starting send_groups_reminder_email task.")
+        lms_client = LmsApiClient()
+        enterprise_catalog_client = EnterpriseCatalogApiClient()
+        policy_group_associations = PolicyGroupAssociation.objects.all()
+        for policy_group_association in policy_group_associations:
+            pecu_email_properties = []
+            enterprise_group_uuid = policy_group_association.enterprise_group_uuid
+            pending_enterprise_customer_users = (
```

Comment:
we'll want to fetch the customer data on this level, using the customer associated with the policy